generators/vsix-vs-win/BotBuilderVSIX-V4/ProjectTemplates/Multi-project/BasicBot/DeploymentScripts/MsbotClone/bot.recipe

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "resources": [
+    {
+      "type": "endpoint",
+      "id": "1",
+      "name": "development",
+      "url": "http://localhost:3978/api/messages"
+    },
+    {
+      "type": "luis",
+      "id": "34",
+      "name": "basic-bot-LUIS"
```

Comment:
>basic-b [](start = 15, length = 7)

I guess if we $safeprojectname$'d this, we'd have to change the constant in the code.  
I'm not sure how the id vs name works in bot.recipe vs .bot file.